src/lter/network_helpers.R

```diff
@@ -39,11 +39,37 @@ get_avail_lter_product_sets <- function(prodname_ms, version, domain, data_track
     element_ids = reqdata[,2]
     dl_urls = paste0(dl_endpoint, domain_ref, '/', prodcode, '/', version,
         '/', element_ids)
-
+    
+    names <- str_match(reqdata[,3], '(.+?)_.*')[,2]
+    
+    #check to see if site information was included in lter informaiton
+    if(all(is.na(names))) {
```

Comment:
here you account for the case where all names are NA, and below you account for the case where no names are NA. what about when some names are NA?